client/src/components/LoginModal.jsx

```diff
@@ -33,10 +34,19 @@ const LoginModal = () => {
         login(data);
         closeLoginModal()
       } else {
-        console.error('Login failed:', data.message);
+        Modal.warning({
+          title: 'Login failed',
+          content:  <div>
+          <p>Invalid username/password. Please check your credentials and try again.</p>
+        </div>,
+          onOk: () => {},
+        });
       }
     } catch (error) {
```

Comment:
Let's be discerning in the kind of error we are catching here. If it is really a network error (you should test this based on error code or some other property within the variable "error"), then the Modal.error you have is applicable. For all other kinds of errors, I think you ca keep the former console.error line for now.

Reply:
The catch error would be outside of any user input or db query errors.

Reply:
I see. Should we log the error in console output too? As developer, you won't be able to debug this user issue in the off-chance it actually occurs because you aren't saving that info anywhere for you to review.

Reply:
[3aaec21](https://github.com/MBHunter88/queer_conceptions/pull/11/commits/3aaec218675c69d221db5dbc785bbc1ee3618207)

Got it, I added a client side console.error to use for debugging, allowing monitoring and helping address issues more reliably.